client/me/concierge/shared/calendar-card.js

```diff
@@ -104,8 +104,8 @@ class CalendarCard extends Component {
 		const date = this.withTimezone( this.props.date );
 
 		return (
-			<div className="concierge__calendar-card-header">
-				<Gridicon icon="calendar" className="concierge__calendar-card-header-icon" />
+			<div className="shared__calendar-card-header">
```

Comment:
Maybe just `calendar-card__header`

Reply:
The parent folder prefix is required for a successful linting validation.